CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ CHANGELOG
 
 **CHANGES**
 - Upgrade Python runtime used by Lambda functions in AWS Batch integration to python3.9.
```

Comment:
I'd list this as the last item

Reply:
Changed. Also update the changelog of EFA to be 1.19.0